packages/common/src/types.ts

```diff
@@ -173,6 +173,11 @@ export interface IHubGeography {
   geometry?: IGeometry;
 }
 
+export type SearchableType = IItem | IGroup | IUser;
+export type SearchFunction = (
+  ...args: any
```

Comment:
Since we're using the `rest` operator, I'd expect this to be

```
export type SearchFunction = (
  ...args: any[]
) => Promise<ISearchResult<SearchableType>>;
```